api/FileError.json

```diff
@@ -0,0 +1,60 @@
+{
+  "api": {
+    "FileError": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/FileError",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": "13",
+            "version_removed": "53",
+            "prefix": "webkit"
+          },
+          "chrome_android": {
+            "version_added": "0.16",
+            "version_removed": "53",
+            "prefix": "webkit"
+          },
+          "edge": {
+            "version_added": null
+          },
+          "edge_mobile": {
+            "version_added": null
+          },
+          "firefox": {
+            "version_added": true,
+            "version_removed": "13",
+            "notes": "The <code>FileError</code> interface has been removed starting with Gecko 13 (Firefox 13.0 / Thunderbird 13.0 / SeaMonkey 2.10). Instead the more general <a href='/en-US/docs/Web/API/DOMError'><code>DOMError</code></a> interface is used and returned by <a href='/en-US/docs/Web/API/FileReader'><code>FileReader.error</code></a>."
```

Comment:
Please rename this note to say ""The <code>FileError</code> interface has been removed starting with Firefox 13. (..)". We no longer want to mention gecko, thunderbird, seamonkey in the docs.